### PR TITLE
MHV-42363 Add backend support for displaying MHV radiology images

### PR DESCRIPTION
**Pull request by mmoyer-va**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

- Added the new pass-through endpoints related to Blue Button Medical Imaging functionality.

### [MHV-42363](https://jira.devops.va.gov/browse/MHV-42363) - Add Download (backend) functionality for Radiology Images ###

> MR to VA.gov: Add download (backend) functionality for Radiology Images
> 
> User Story: As a Medical Records User, I want to download my Radiology Images so I have it on my computer.
> 
> Acceptance Criteria:
> AC1 The ability to download Radiology Images has been implemented
> AC2
> AC3
> AC4
> AC5
> 
> Links to UCD:
> Sketch Link:
> 3/28/23 Still working on design--will update once designs are complete.
> Other Design Notes
> 
> Architectural Notes: Always check for the latest designs in Sketch!

## Testing done

- Added spec tests for the new functions

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
MHV Medical Records

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog (if applicable)
- [x]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

**Review by mattwrightva — DISMISSED** [review]
Nice work!

File: spec/lib/medical_records/bb_internal/client_spec.rb [hunk 1]
```diff
@@ -6,20 +6,19 @@
...
   before(:all) do
-    VCR.use_cassette 'mr_client/bb_internal/session' do
-      VCR.use_cassette 'mr_client/bb_internal/get_patient' do
-        @client ||= begin
-          client = BBInternal::Client.new(session: { user_id: '15176497' })
-          client.authenticate
-          client
-        end
+    # The "get_patient" cassette also contains the session auth call.
+    VCR.use_cassette 'mr_client/bb_internal/get_patient', record: :new_episodes do
```
Comment (stevenjcumming) [inline spec/lib/medical_records/bb_internal/client_spec.rb:10]: Can you remove `record: :new_episodes` please

**Review by stevenjcumming — CHANGES_REQUESTED** [review]

File: spec/lib/medical_records/bb_internal/client_spec.rb (hunk 1 above)
Comment (mmoyer-va) [reply spec/lib/medical_records/bb_internal/client_spec.rb]: @stevenjcumming good catch! I removed it.

**Review by mattwrightva — APPROVED** [review]
Looks good!

**Review by stevenjcumming — APPROVED** [review]

**Pull request merged**